src/core/filter/security.go

```diff
@@ -110,6 +111,7 @@ func Init() {
 	// standalone
 	reqCtxModifiers = []ReqCtxModifier{
 		&secretReqCtxModifier{config.SecretStore},
+		&oidcCliReqCtxModifier{},
```

Comment:
we should notice that all of these modifiers are not in VIC mode.

Reply:
This is expected because in VIC the auth is delegated to admiral.